index.bs

```diff
@@ -408,14 +408,27 @@ Excluding inputs generally include
 However, an event whose only effect is to begin or update a fling or scroll
 gesture is not an excluding input.
 
-The user agent may delay the reporting of layout shifts after a
-<a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event
-until such time as it is known that the event does not begin a fling or scroll
-gesture.
+The user agent buffers the layout shifts of
+<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a> events after a
```

Comment:
What is "the layout shifts of pointmove events"? This is not super clear to me (layout shifts don't belong to inputs).

Reply:
I also thought we agree that we do not need to care about move events since the input exclusions can be determined solely from the downs, ups, and cancels? Did this change in the implementation?

Reply:
Do you mean we do not need to specify whether pointermove events or mousemove events are excluding inputs or not.